app/transformers/ATSCalculations.scala

```diff
@@ -31,23 +31,21 @@ trait ATSCalculations extends DoubleUtils with Logging {
   val taxRates: TaxRateService
   val incomeTaxStatus = summaryData.incomeTaxStatus
 
-  def get(liability: Liability): Amount = {
-    val logger = Logger(getClass.getName)
+  def get(liability: Liability): Amount =
     summaryData.atsData.getOrElse(
       liability,
       summaryData.nationalInsuranceData.getOrElse(liability, {
-        logger.info(s"Unable to retrieve $liability")
-        throw ATSParsingException(liability.apiValue)
+        val ex = ATSParsingException(liability.apiValue)
+        logger.error(s"Unable to retrieve $liability", ex)
```

Comment:
was this meant to be changed to error?

Reply:
That one yes it should at error level. I fixed the issue I had last time with getwithdefault which was trying on get....